nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpResponse.java

```diff
@@ -155,9 +155,9 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
         try {
             session.exportTo(flowFile, response.getOutputStream());
             response.flushBuffer();
-        } catch (final IOException ioe) {
+        } catch (final Exception e) {
```

Comment:
Instead of changing this to blanket catching "Exception". Could another catch block that handles "FlowFileAccessException" be added, or just add it as another exception type caught in this block? 

Blanket catching Exception is considered bad form so unless there is a specific need for it, I tend to stay away from it.